src/composable/copy.ts

```diff
@@ -7,7 +7,7 @@ export function useCopy({ source, text = 'Copied to the clipboard' }: { source?:
 
   return {
     async copy(content?: string, { notificationMessage }: { notificationMessage?: string } = {}) {
-      await copy();
+      await copy(content);
```

Comment:
This causes breaking functionality, for instance you can verify on UUIDs: you will receive [object PointerEvent]